src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -0,0 +1,24 @@
+Arg = {...}
+
+if not Arg[1] then
+  print("Usage: webrun <url> <args>")
+  return
+end
+
+if not http then
+print("Please enable http in the config")
+return
+end
+
+local test,text = http.checkURL(Arg[1])
```

Comment:
Just stylistic, but please add whitespaces between the commas (`local test,text` -> `local test, text`)